components/selectors/parser.rs

```diff
@@ -1201,6 +1201,8 @@ pub mod tests {
         type PseudoElement = PseudoElement;
     }
 
+    type ShutUpTidy = String;
```

Comment:
I see. You can probably do `&<Self::Impl as SelectorImpl>::NamespacePrefix` though.